erc/ERCS/erc-7562.md

```diff
@@ -180,42 +272,42 @@ To help make sense of these params, note that a malicious paymaster can at most
     * **[OP-054]** Any other access to the `EntryPoint` is forbidden
 * `*CALL` opcodes:
     * **[OP-061]** `CALL` with `value` is forbidden. The only exception is a call to the `EntryPoint` described above.
     * **[OP-062]** Precompiles:
         * Only allowed the core 9 precompiles.\
-          Specifically the computation precompiles that do not access anything in the blockchain state or environment.
-* **[OP-070] Transient Storage slots defined in [EIP-1153](./eip-1153.md) and accessed using `TLOAD` (`0x5c`) and `TSTORE` (`0x5d`) opcodes
-    are treated exactly like persistent storage (SLOAD/SSTORE)
+          Specifically, the computation precompiles that do not access anything in the blockchain state or environment.
+* **[OP-070]** Transient Storage slots defined in [EIP-1153](./eip-1153) and accessed using `TLOAD` (`0x5c`) and `TSTORE` (`0x5d`) opcodes
+  are treated exactly like persistent storage (SLOAD/SSTORE)
 
 ### Code Rules
 
 * **[COD-010]** Between the first and the second validations, the `EXTCODEHASH` value of any visited address,
-entity or referenced library, may not be changed.\
-If the code is modified, the UserOperation is considered invalid.
+  entity, or referenced library, may not be changed.\
+  If the code is modified, the UserOperation is considered invalid.
 
 ### Storage rules.
 
-The permanent storage access with `SLOAD` and `SSTORE` instructions within each phase are limited as follows:
+The permanent storage access with `SLOAD` and `SSTORE` instructions within each phase is limited as follows:
 
 * **[STO-010]** Access to the "account" storage is always allowed.
 * Access to associated storage of the account in an external (non-entity contract) is allowed if either:
     * **[STO-021]**  The account already exists
     * **[STO-022]**  There is an `initCode` and the `factory` contract is staked.
 * If the entity (`paymaster`, `factory`) is staked, then it is also allowed:
     * **[STO-031]** Access the entity's own storage.
-    * **[STO-032]** Read/Write Access to storage slots that is associated with the entity, in any non-entity contract.
+    * **[STO-032]** Read/Write Access to storage slots that are associated with the entity, in any non-entity contract.
     * **[STO-033]** Read-only access to any storage in non-entity contract.
 * **[STO-040]** `UserOperation` may not use an entity address (`factory`/`paymaster`/`aggregator`) that is used as an "account" in another `UserOperation` in the mempool. \
-This means that a `Paymaster` and `Factory` contracts cannot practically be an "account" contract as well.
+  This means that `Paymaster` and `Factory` contracts cannot practically be an "account" contract as well.
 * **[STO-041]** A contract whose Associated Storage slot is accessed during a `UserOperation` may not be an address of a "sender" in another `UserOperation` in the mempool.
 
 ### Staked Entities Reputation Rules
```

Comment:
Oddly my reply to this seem to have disappeared, and the comment was resolved.  Unresoving because this rule still says the opposite of what we intended to do.  It limits the sender, not the contract that attacks it by using it in associated storage.  I previously wrote a detailed explanation which is apparently lost somehow.

Reply:
The way I'd interpret the rule as written above:
"If the mempool has op1 and op2, and op1 accesses someContract.balances[op2.sender], exclude op2".
which allows op1 to censor op2.

The intended behavior is the opposite:
"If the mempool has op1 and op2, and op1 accesses someContract.balances[op2.sender], exclude op1".